### ISSUE TITLE
Stack commands fail with "ResolveEndpointV2" error message since 1.7.4

### ISSUE THREAD

**Issue by vjeantet** [aws-cloudformation/rain#232]
Hello, since 1.7.4 all commands which require to connect to cloudformation fail with a red message 
* "failed to get logs for stack.... : not found, ResolveEndpointV2"
* "error watching stack ... : not found, ResolveEndpointV2"
* "failed to list stacks: not found, ResolveEndpointV2"

If i run the same commands with the 1.7.3 or older, it works.

Bellow a capture of commands run with 1.7.4 and 1.7.3

```bash
$rain-1.7.4 -ls --debug
DEBUG: Loading AWS config
DEBUG: Fetching stacks in eu-central-1
panic: failed to list stacks: not found, ResolveEndpointV2 [recovered]
	panic: failed to list stacks: not found, ResolveEndpointV2
 
goroutine 1 [running]:
github.com/aws-cloudformation/rain/internal/cmd.execute.func1()
	github.com/aws-cloudformation/rain/internal/cmd/wrap.go:77 +0x168
panic({0x1045263c0?, 0x14000864ba0?})
	runtime/panic.go:914 +0x218
github.com/aws-cloudformation/rain/internal/cmd/ls.glob..func1(0x140007e6f00?, {0x14000880990?, 0x4?, 0x102f323b5?})
	github.com/aws-cloudformation/rain/internal/cmd/ls/ls.go:62 +0x9dc
github.com/spf13/cobra.(*Command).execute(0x104adb320, {0x14000880970, 0x1, 0x1})
	github.com/spf13/cobra@v1.8.0/command.go:987 +0x828
github.com/spf13/cobra.(*Command).ExecuteC(0x104ad8dc0)
	github.com/spf13/cobra@v1.8.0/command.go:1115 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/aws-cloudformation/rain/internal/cmd.execute(0x1022fd414?)
	github.com/aws-cloudformation/rain/internal/cmd/wrap.go:86 +0x50
github.com/aws-cloudformation/rain/internal/cmd.Execute(0x104a99d78?)
	github.com/aws-cloudformation/rain/internal/cmd/wrap.go:95 +0x1c
main.main()
	./main.go:23 +0x24
 
 
$ rain-1.7.3 ls --debug
DEBUG: Loading AWS config
DEBUG: Fetching stacks in eu-central-1
CloudFormation stacks in eu-central-1:
  StackSet-**********: CREATE_COMPLETE
  StackSet-****************************************: UPDATE_COMPLETE
```

**Comment by vjeantet** [comment]
I don't know if it helps, but i found this issue in the go-sdk projects, with this comment marked as a solution : 
* https://github.com/aws/aws-sdk-go-v2/issues/2370#issuecomment-181490338

**Comment by ericzbeard** [comment]
I will have a new release out soon. Needed to update all sdk dependencies to the latest versions.

**Comment by ericzbeard** [comment]
Fixed in v1.7.5. #233
